core/controllers/topic_editor_test.py

```diff
@@ -70,6 +77,71 @@ def test_story_creation(self):
         self.logout()
 
 
+class SubtopicPageEditorTest(BaseTopicEditorControllerTest):
+
+    def test_editable_subtopic_page_get(self):
+        # Check that non-admins cannot access the editable subtopic data.
```

Comment:
Does "non-admins" include topic managers? That seems weird.

For thoroughness please add a check for topic managers, too.

Reply:
Done, added tests.